<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add</class>
 <widget class="QWidget" name="Add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>40</horstretch>
     <verstretch>40</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 213, 129) ;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>30</width>
     <height>29</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>30</horstretch>
     <verstretch>30</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:rgba(225,100,100,30);
	border-style:outset;                 
	border-width:4px;                    
	border-radius:10px;                  
	border-color:rgba(255,255,255,30);   
	font:bold 10px;                      
	color:rgba(0,0,0,100);
	padding:6px; 
}

QPushButton:pressed{
	background-color:rgba(225,100,100,200);
	border-color:rgba(255,255,255,30);
	border-style:inset;
	color:rgba(0,0,0,100);
}
//鼠标悬停样式
QPushButton:hover
{
	background-color:rgba(255,100,100,100);
	border-color:rgba(255,255,255,200);
	color:rgba(0,0,0,200);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/prefix1/resource/cc-close-crude.png</normaloff>:/new/prefix1/resource/cc-close-crude.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>401</width>
     <height>318</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QLineEdit" name="nameEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">padding:1px;
font:1000 15pt &quot;黑体&quot;;  </string>
      </property>
      <property name="text">
       <string>事项：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">padding:1px;
font:1000 15pt &quot;黑体&quot;;  </string>
      </property>
      <property name="text">
       <string>时间：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="styleSheet">
           <string notr="true">QCheckBox{
	border: none;
}
QCheckBox::indicator{
	background-color: rgba(255, 255, 255, 0);
	border: 0px solid #b1b1b1;
	width: 25px;
	height: 25px;
}
QCheckBox:enabled:checked{
	color: rgb(255, 255, 255);
}
QCheckBox:enabled:!checked{
	color: rgb(255, 255, 255);
}
QCheckBox:enabled:hover{
	color: rgb(0, 200, 0);
}
QCheckBox:!enabled{
	color: rgb(80, 80, 80);
}
QCheckBox::indicator:unchecked {
	image: url(:/new/prefix1/resource/checkbox-blank-circle-outline .png);
}

QCheckBox::indicator:checked {
	image: url(:/new/prefix1/resource/checkbox-marked-circle-outline.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">padding:1px;
font:1000 15pt &quot;黑体&quot;;  </string>
          </property>
          <property name="text">
           <string>周一</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="checkBox_2">
          <property name="styleSheet">
           <string notr="true">QCheckBox{
	border: none;
}
QCheckBox::indicator{
	background-color: rgba(255, 255, 255, 0);
	border: 0px solid #b1b1b1;
	width: 25px;
	height: 25px;
}
QCheckBox:enabled:checked{
	color: rgb(255, 255, 255);
}
QCheckBox:enabled:!checked{
	color: rgb(255, 255, 255);
}
QCheckBox:enabled:hover{
	color: rgb(0, 200, 0);
}
QCheckBox:!enabled{
	color: rgb(80, 80, 80);
}
QCheckBox::indicator:unchecked {
	image: url(:/new/prefix1/resource/checkbox-blank-circle-outline .png);
}

QCheckBox::indicator:checked {
	image: url(:/new/prefix1/resource/checkbox-marked-circle-outline.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">padding:1px;
font:1000 15pt &quot;黑体&quot;;  </string>
          </property>
          <property name="text">
           <string>周二</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QCheckBox" name="checkBox_3">
          <property name="styleSheet">
           <string notr="true">QCheckBox{
	border: none;
}
QCheckBox::indicator{
	background-color: rgba(255, 255, 255, 0);
	border: 0px solid #b1b1b1;
	width: 25px;
	height: 25px;
}
QCheckBox:enabled:checked{
	color: rgb(255, 255, 255);
}
QCheckBox:enabled:!checked{
	color: rgb(255, 255, 255);
}
QCheckBox:enabled:hover{
	color: rgb(0, 200, 0);
}
QCheckBox:!enabled{
	color: rgb(80, 80, 80);
}
QCheckBox::indicator:unchecked {
	image: url(:/new/prefix1/resource/checkbox-blank-circle-outline .png);
}

QCheckBox::indicator:checked {
	image: url(:/new/prefix1/resource/checkbox-marked-circle-outline.png);
}QCheckBox{
	border: none;
}
QCheckBox::indicator{
	background-color: rgba(255, 255, 255, 0);
	border: 0px solid #b1b1b1;
	width: 25px;
	height: 25px;
}
QCheckBox:enabled:checked{
	color: rgb(255, 255, 255);
}
QCheckBox:enabled:!checked{
	color: rgb(255, 255, 255);
}
QCheckBox:enabled:hover{
	color: rgb(0, 200, 0);
}
QCheckBox:!enabled{
	color: rgb(80, 80, 80);
}
QCheckBox::indicator:unchecked {
	image: url(:/new/prefix1/resource/checkbox-blank-circle-outline .png);
}

QCheckBox::indicator:checked {
	image: url(:/new/prefix1/resource/checkbox-marked-circle-outline.png);
}QCheckBox{
	border: none;
}
QCheckBox::indicator{
	background-color: rgba(255, 255, 255, 0);
	border: 0px solid #b1b1b1;
	width: 25px;
	height: 25px;
}
QCheckBox:enabled:checked{
	color: rgb(255, 255, 255);
}
QCheckBox:enabled:!checked{
	color: rgb(255, 255, 255);
}
QCheckBox:enabled:hover{
	color: rgb(0, 200, 0);
}
QCheckBox:!enabled{
	color: rgb(80, 80, 80);
}
QCheckBox::indicator:unchecked {
	image: url(:/new/prefix1/resource/checkbox-blank-circle-outline .png);
}

QCheckBox::indicator:checked {
	image: url(:/new/prefix1/resource/checkbox-marked-circle-outline.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">padding:1px;
font:1000 15pt &quot;黑体&quot;;  </string>
          </property>
          <property name="text">
           <string>周三</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QCheckBox" name="checkBox_4">
          <property name="styleSheet">
           <string notr="true">QCheckBox{
	border: none;
}
QCheckBox::indicator{
	background-color: rgba(255, 255, 255, 0);
	border: 0px solid #b1b1b1;
	width: 25px;
	height: 25px;
}
QCheckBox:enabled:checked{
	color: rgb(255, 255, 255);
}
QCheckBox:enabled:!checked{
	color: rgb(255, 255, 255);
}
QCheckBox:enabled:hover{
	color: rgb(0, 200, 0);
}
QCheckBox:!enabled{
	color: rgb(80, 80, 80);
}
QCheckBox::indicator:unchecked {
	image: url(:/new/prefix1/resource/checkbox-blank-circle-outline .png);
}

QCheckBox::indicator:checked {
	image: url(:/new/prefix1/resource/checkbox-marked-circle-outline.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="styleSheet">
           <string notr="true">padding:1px;
font:1000 15pt &quot;黑体&quot;;  </string>
          </property>
          <property name="text">
           <string>周四</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QCheckBox" name="checkBox_5">
          <property name="styleSheet">
           <string notr="true">QCheckBox{
	border: none;
}
QCheckBox::indicator{
	background-color: rgba(255, 255, 255, 0);
	border: 0px solid #b1b1b1;
	width: 25px;
	height: 25px;
}
QCheckBox:enabled:checked{
	color: rgb(255, 255, 255);
}
QCheckBox:enabled:!checked{
	color: rgb(255, 255, 255);
}
QCheckBox:enabled:hover{
	color: rgb(0, 200, 0);
}
QCheckBox:!enabled{
	color: rgb(80, 80, 80);
}
QCheckBox::indicator:unchecked {
	image: url(:/new/prefix1/resource/checkbox-blank-circle-outline .png);
}

QCheckBox::indicator:checked {
	image: url(:/new/prefix1/resource/checkbox-marked-circle-outline.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="styleSheet">
           <string notr="true">padding:1px;
font:1000 15pt &quot;黑体&quot;;  </string>
          </property>
          <property name="text">
           <string>周五</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QCheckBox" name="checkBox_6">
          <property name="styleSheet">
           <string notr="true">QCheckBox{
	border: none;
}
QCheckBox::indicator{
	background-color: rgba(255, 255, 255, 0);
	border: 0px solid #b1b1b1;
	width: 25px;
	height: 25px;
}
QCheckBox:enabled:checked{
	color: rgb(255, 255, 255);
}
QCheckBox:enabled:!checked{
	color: rgb(255, 255, 255);
}
QCheckBox:enabled:hover{
	color: rgb(0, 200, 0);
}
QCheckBox:!enabled{
	color: rgb(80, 80, 80);
}
QCheckBox::indicator:unchecked {
	image: url(:/new/prefix1/resource/checkbox-blank-circle-outline .png);
}

QCheckBox::indicator:checked {
	image: url(:/new/prefix1/resource/checkbox-marked-circle-outline.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="styleSheet">
           <string notr="true">padding:1px;
font:1000 15pt &quot;黑体&quot;;  </string>
          </property>
          <property name="text">
           <string>周六</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QCheckBox" name="checkBox_7">
          <property name="styleSheet">
           <string notr="true">QCheckBox{
	border: none;
}
QCheckBox::indicator{
	background-color: rgba(255, 255, 255, 0);
	border: 0px solid #b1b1b1;
	width: 25px;
	height: 25px;
}
QCheckBox:enabled:checked{
	color: rgb(255, 255, 255);
}
QCheckBox:enabled:!checked{
	color: rgb(255, 255, 255);
}
QCheckBox:enabled:hover{
	color: rgb(0, 200, 0);
}
QCheckBox:!enabled{
	color: rgb(80, 80, 80);
}
QCheckBox::indicator:unchecked {
	image: url(:/new/prefix1/resource/checkbox-blank-circle-outline .png);
}

QCheckBox::indicator:checked {
	image: url(:/new/prefix1/resource/checkbox-marked-circle-outline.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="styleSheet">
           <string notr="true">padding:1px;
font:1000 15pt &quot;黑体&quot;;  </string>
          </property>
          <property name="text">
           <string>周日</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="3">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer_9">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTimeEdit" name="timeEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">padding:1px;
font:1000 15pt &quot;黑体&quot;;  </string>
        </property>
        <property name="text">
         <string>~</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEdit_2"/>
      </item>
     </layout>
    </item>
    <item row="4" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="styleSheet">
         <string notr="true">padding:1px;
font:1000 15pt &quot;黑体&quot;;  </string>
        </property>
        <property name="text">
         <string>优先级：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
